iwyu_ast_util.cc

```diff
@@ -963,28 +968,38 @@ map<const Type*, const Type*> GetTplTypeResugarMapForFunction(
   }
 
   for (const Type* type : fn_arg_types) {
-    // See if any of the template args in retval are the desugared form of us.
+    // See if any of the template args in resugar_map are the desugared form
+    // of us.
     const Type* desugared_type = GetCanonicalType(type);
     if (ContainsKey(desugared_types, desugared_type)) {
-      retval[desugared_type] = type;
+      resugar_map[desugared_type] = type;
       if (desugared_type != type) {
         VERRS(6) << "Remapping template arg of interest: "
                  << PrintableType(desugared_type) << " -> "
                  << PrintableType(type) << "\n";
       }
     }
+    // TODO(bolshakov): more fine-grained determination of provided types could
+    // e.g. consider only arguments that actually take part in type deduction.
+    // Simple moving the line below into the if-statement body above doesn't
```

Comment:
Typo: Simple -> Simply

Reply:
I used "simple" as an adjective to "moving", not an adverb, but I can be wrong, of course.

Reply:
Then I would have expected "Simple moving **_of_** the line...". But my Swedish English is not exactly authoritative :).

I think the "simply" wording is simpler in either case.